web/src/TopBar.svelte

```diff
@@ -37,52 +37,84 @@
   let stages = { ...tierLabels, assessment: "Assess" };
 </script>
 
-<div>
-  <nav aria-label="breadcrumb">
-    <ul>
-      <li>NPW</li>
-      <li><a href="index.html">Select area</a></li>
+<header>
+  <div class="site-navigation">
+    <nav class="ds_site-navigation">
+      <ul class="ds_site-navigation__list">
+        <li class="ds_site-navigation__item">
+          <a
+            class="ds_site-navigation__link"
+            href="https://www.npt.scot/"
+            target="_blank"
+          >
+            <img id="logo" src={logo} alt="NPT logo" />
+            NPW &nbsp;&gt;
+          </a>
+        </li>
 
-      <li><ManageFiles /></li>
+        <li class="ds_site-navigation__item">
+          <a class="ds_site-navigation__link" href="index.html">
+            Select area &nbsp;&gt;
+          </a>
+        </li>
 
-      {#each Object.entries(stages) as [stage, label]}
-        <li>
-          {#if $currentStage == stage}
-            <b>{label}</b>
-          {:else}
-            <Link on:click={() => changeStage(stage)}>{label}</Link>
-          {/if}
+        <li class="ds_site-navigation__item">
+          <ManageFiles />
         </li>
-      {/each}
-    </ul>
-  </nav>
+
+        {#each Object.entries(stages) as [stage, label]}
+          <li class="ds_site-navigation__item">
+            <!-- svelte-ignore a11y-invalid-attribute -->
+            <a
+              class="ds_site-navigation__link {stage}"
+              class:ds_current={$currentStage == stage}
+              href="#"
+              on:click|preventDefault={() => changeStage(stage)}
+            >
+              {label}
+              {#if stage != "assessment"}
+                &nbsp;&gt;
+              {/if}
+            </a>
+          </li>
+        {/each}
+      </ul>
+    </nav>
+  </div>
 
   <TopBarStats />
-</div>
+</header>
 
 <style>
-  div {
+  header {
     display: flex;
-    justify-content: space-between;
-    padding: 4px;
+    padding: 0 20px;
+    box-shadow: 0px 10px 10px 0px #eee;
   }
-
-  nav {
-    padding: 0 0.5rem;
+  header #logo {
+    height: 30px;
+    margin-right: 10px;
+    vertical-align: middle;
   }
 
-  nav ul {
-    display: flex;
-    flex-wrap: wrap;
-    list-style: none;
-    margin: 0;
-    padding: 0;
-    align-items: end;
+  .ds_site-navigation .ds_site-navigation__link {
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+  .ds_site-navigation .ds_current {
+    font-weight: bold;
   }
 
-  nav li:not(:last-child)::after {
-    display: inline-block;
-    margin: 0 0.25rem;
-    content: ">";
+  .ds_site-navigation .ds_site-navigation__link.Primary:not(:focus):after {
```

Comment:
If you're following the commits, my strategy was to first copy in all the CSS to `style.css`, applied globally. Much of it had no effect, based on the selectors not being set up. Then I went to each part of the page and made it match the intended page, usually by moving the relevant styles to the Svelte component. Unused rules get warnings when they're not global styles, which is super nice.